src/set_theory/cardinal/basic.lean

```diff
@@ -621,6 +621,15 @@ theorem sum_const' (ι : Type u) (a : cardinal.{u}) : sum (λ _:ι, a) = #ι * a
 by simpa only [mk_sigma, mk_sum, mk_out, lift_id] using
   mk_congr (equiv.sigma_sum_distrib (quotient.out ∘ f) (quotient.out ∘ g))
 
+@[simp] theorem sum_add_distrib' {ι} (f g : ι → cardinal) :
+  cardinal.sum (λ i, f i + g i) = sum f + sum g :=
```

Comment:
I wonder if this is more useful with
```suggestion
  cardinal.sum (λ i, f i + g i) = sum (λ i, f i) + sum (λ i, g i) :=
```
which keeps the lambda around if you had it in the first place

Reply:
I don't see why we should favor using lambdas here and not anywhere else.

Reply:
I don't like Eric's suggestion, if `f = λ j, ...`, then his suggestion will give `sum (λ i, (λ j, ...) i)